leap_c/env_wrapper.py

```diff
@@ -0,0 +1,18 @@
+"""Provides logic that allows to wrap environments with additional functionality."""
+
+from typing import Any, SupportsFloat
+import gymnasium as gym
+
+
+class ActionStatsWrapper(gym.Wrapper):
+    """A wrapper that logs the actions taken by the agent."""
+
+    def step(self, action) -> tuple[Any, SupportsFloat, bool, bool, dict[str, Any]]:
+        *transition, info = super().step(action)
```

Comment:
Very nice syntax, I oftentimes needed something like this, but didn't bother to look it up. Thanks, definitely will try :D